### PR TITLE
Determinator blog post

### PR DESCRIPTION
**Pull request by Greyschale**
Growth team's exciting outing, and how we randomised the scavenger hunt teams.

🎉

### CODE REVIEW

**Comment by krissy** [comment]
This post is super exciting for many reasons, including sloths! 👌  Was the team photo going to be in here?

**Comment by Greyschale** [comment]
@krissy I thought about adding the group photo, but considering not everybody responded with the OK to post it, I figured it'd be safest to leave it out for now. But we could definitely add the one from the repo, @jphastings !

File: _posts/2017-05-10-determinating-a-scavenger-hunt.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+| ---------- | -------- |
+| Red Pandas | 01:11:08 |
+| Giraffes   | 01:37:17 |
+| Sloths     | 02:06:10 |
+
+Congratulations, Red Pandas! 🎉 And...
+
+## We'll Be Back.
+
+![The Determinator](/images/posts/determinating-scavenger-hunt/determinator.jpg)
```
Comment (kapowaz) [inline _posts/2017-05-10-determinating-a-scavenger-hunt.md]: If you could wrap this inside a `<figure></figure>` element, it’ll ensure it lays out correctly at all the various breakpoints.

File: _posts/2017-05-10-determinating-a-scavenger-hunt.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+For our scavenger hunt, we ended up with 3 teams of 6 people each, all randomly assigned (minus captains, for an unrelated reason). All that remained was to solve some puzzles.
+
+## The Data Could be Called 'Results'
+
+In a thrilling battle of wits, Teams Red Panda, Giraffe, and Sloth trekked through London City chasing puzzle after puzzle, with some well-deserved breaks in between. Some of the hints were tricky, but all the teams managed to complete everything by the combined brain powers of the Growth Team.
+
+Who won in the end? Team Red Pandas eked out a close win against the rest.
+
+| Team       | Time     |
```
Comment (kapowaz) [inline _posts/2017-05-10-determinating-a-scavenger-hunt.md]: The CSS for this isn’t being picked up (it’s only applied to tables inside `<article class="guidelines">` for guidelines pages, but it needn’t be that specific) — I’ve got a commit to fix this, I’ll push it up once you give me the all-clear!

File: _posts/2017-05-10-determinating-a-scavenger-hunt.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+| ---------- | -------- |
+| Red Pandas | 01:11:08 |
+| Giraffes   | 01:37:17 |
+| Sloths     | 02:06:10 |
+
+Congratulations, Red Pandas! 🎉 And...
```
Comment (kapowaz) [inline _posts/2017-05-10-determinating-a-scavenger-hunt.md]: For some reason the emoji here isn’t being given the correct spacing:

img

…one possible fix is to insert a couple of non-breaking spaces between in and the word _And_, like:

```
🎉&nbsp;&nbsp;And...
```

I leave it up to you to decide if this is important or not!

File: _posts/2017-05-10-determinating-a-scavenger-hunt.md (hunk 2 above)
Comment (kapowaz) [inline _posts/2017-05-10-determinating-a-scavenger-hunt.md]: I’ve pushed this commit up now 👍

**Pull request merged**